chibuezeayogu/Library-App
# #163680634 Implement User signup

**What does the PR do?**
- This PR implements user `sign up`

**Description of Task to be completed?**
- Ensure the users can sign up
- Ensure that token is generated when users sign's up 
- Ensure test is passing 

**How should this be manually tested?**
- Clone the repository [Library-App](https://github.com/chibuezeayogu/Library-App.git) and
- cd into `Events-Manager` and checkout to  this branch `ft-implement-user-signup-163680634`
- run `bundle install` to install new `rails gems`
- run `rails db:migrate` for both `development` and `test` environments
- run `rails server` and test this route ` Post /api/vi/users/signup` using `Postman`
- run `rspec` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#163680634](https://www.pivotaltracker.com/story/show/163680634)
